### ISSUE TITLE
JInternalFrame resize hides frame's glass pane

### ISSUE THREAD

**Issue by mgarin** [mgarin/weblaf#461]
In `BasicInternalFrameUI.BorderListener` there is next piece of code:
```java
Container c = frame.getTopLevelAncestor ();
if ( c instanceof RootPaneContainer )
{
    Component glassPane = ( ( RootPaneContainer ) c ).getGlassPane ();
    glassPane.setCursor ( Cursor.getPredefinedCursor ( Cursor.DEFAULT_CURSOR ) );
    glassPane.setVisible ( false );
}
```
I looked through different implementations of the `BasicInternalFrameUI` and this part of code seems to be a workaround for the common proxy drag/resize usage case. Thought it is completely useless for WebLaF and even harms some of the components which use glass pane.

There is no way to properly replace `BorderListener` with a custom implementation due to all fields being private or package local, so I'll have to fully replace UI class with my own similar implementation.

**Comment by mgarin** [comment]
I was pretty sure this got fixed along with the rework for related components, but apparently issue is still there and requires a fix. I will be adding it in v1.2.10 update.

**Comment by mgarin** [comment]
Unfortunately there is no easy way to fix the problem without replacing `BasicInternalFrameUI` because internal Swing code is messy and full of private/package local variables and methods which cannot be easily overridden to fix issues. 

That is why I am going to make my own implementation of the basic UI, it is a long-term plan for all components anyway.

**Comment by mgarin** [comment]
Added a separate note for changing custom tooltip display from glass pane to layered pane in #520

**Comment by mgarin** [comment]
I'm working on #523 right now, this problem will also go once those changes are available.

**Comment by mgarin** [comment]
`JDesktopPane`, `JInternalFrame` and `JDesktopIcon` revamps have been pushed into `master` branch and will be available in v1.2.10 update. This issue was also fixed in the reworked UI version.
